src/main/kotlin/com/coder/gateway/CoderRemoteConnectionHandle.kt

```diff
@@ -523,5 +522,25 @@ class CoderRemoteConnectionHandle {
 
     companion object {
         val logger = Logger.getInstance(CoderRemoteConnectionHandle::class.java.simpleName)
+        fun processSetupCommand(
+            output: () -> String,
```

Comment:
minor: this is more like the command to execute, so.. "execCommand" is probably a better name!?